liquibase-core/src/main/java/liquibase/LiquibaseRunner.java

```diff
@@ -0,0 +1,225 @@
+package liquibase;
+
+import liquibase.changelog.annotations.reader.ChangeEntry;
+import liquibase.changelog.annotations.reader.ChangeService;
+import liquibase.database.GenericDataStore;
+import liquibase.exception.LiquibaseChangeSetException;
+import liquibase.exception.LiquibaseConfigurationException;
+import liquibase.exception.LiquibaseException;
+import liquibase.logging.LogService;
+import liquibase.logging.Logger;
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.context.ApplicationContext;
+import org.springframework.core.env.Environment;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.List;
+
+import static org.springframework.util.StringUtils.hasText;
+
+public class LiquibaseRunner implements InitializingBean {
+
```

Comment:
Main class responsible for running all changeset. Similar to the Liquicouch class https://github.com/deniswsrosa/liquicouch

Reply:
I think this class would correspond to liquibase.integration.commandline.Main ? We should be able to just use that once you have all the support in, correct?

Reply:
@nvoxland  It was designed initially to be called during the application startup (https://github.com/deniswsrosa/liquicouch-demo/blob/master/src/main/java/com/cb/springdata/sample/SampleApplication.java), but we can add support for command line as well.